src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -2419,6 +2419,13 @@ public ObjectId getHeadRev(String url, String branchSpec) throws GitException, I
     /** {@inheritDoc} */
     public Map<String, ObjectId> getRemoteReferences(String url, String pattern, boolean headsOnly, boolean tagsOnly)
             throws GitException, InterruptedException {
+        return getRemoteReferences(url, pattern != null ? Collections.singletonList(pattern) : Collections.EMPTY_LIST,
+                headsOnly, tagsOnly);
+    }
+
+    /** {@inheritDoc} */
```

Comment:
I argue that you should always add `/** {@inheritDoc} */` to record the fact that the contract has been retained as inherited. Otherwise with no comment present then you do not know if the contract has been changed or retained... inaction is not the way to record a documentation decision.

So 

* leave blank if you have not decided whether the inherited contract has been changed or not **yet**
* put `/** {@inheritDoc} */` to indicate that the contract is exactly as inherited
* put the new contract in a comment to indicate that the contract has been changed (narrowed is the preference but it may be widened.